integrations/twilio/webhooks/auth.go

```diff
@@ -43,12 +46,17 @@ func (h handler) HandleAuth(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	at := ""
+
 	accountSID := r.Form.Get("account_sid")
 	username := accountSID
 	password := r.Form.Get("auth_token")
 	if password == "" {
 		username = r.Form.Get("api_key")
 		password = r.Form.Get("api_secret")
+		at = integrations.APIKey
+	} else {
+		at = integrations.APIToken
```

Comment:
More idiomatic Go style: remove the `else` branch completely, and assign `at := integrations.APIToken` in line 49.

Reply:
Done